pkgs/development/libraries/gstreamer/good/default.nix

```diff
@@ -48,7 +48,7 @@ assert raspiCameraSupport -> (stdenv.isLinux && stdenv.isAarch64);
 
 stdenv.mkDerivation rec {
   pname = "gst-plugins-good";
-  version = "1.18.4";
+  version = "1.18.5";
 
   outputs = [ "out" "dev" ];
 
```

Comment:
Hash did not get an update?

Reply:
Strange, not sure why it would build on my machine if the hash is wrong (perhaps if it's pre-fetched it bypasses some check?). I've updated the hash.